commit_check/__init__.py

```
@@ -32,7 +32,8 @@
         },
         {
             'check': 'author_name',
-            'regex': r'^[A-Za-z ,.\'-]+$|.*(\[bot])',
+            'regex': r'^[A-Za-zÀ-ÖØ-öø-ÿĀ-ž\u0100-\u017F\u0180-\u024F ,.\'-]+$|.*(\[bot])',
+            'regex': r'^[A-Za-zÀ-ÖØ-öø-ÿ\u0100-\u017F\u0180-\u024F ,.\'-]+$|.*(\[bot])',
```

Ruff (0.8.2)

Dictionary key literal `'regex'` repeated

(F601)